cleaner/podman/cleaner.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2022.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package podman
+
+import (
+	"context"
+	"os"
+
+	"github.com/containers/podman/v4/pkg/bindings"
+	"github.com/containers/podman/v4/pkg/bindings/images"
+	"github.com/containers/podman/v4/pkg/domain/entities"
+	"github.com/sirupsen/logrus"
+)
+
+// A valid URI connection should be scheme://
+// For example tcp://localhost:<port>
+// or unix:///run/podman/podman.sock
+// or ssh://<user>@<host>[:port]/run/podman/podman.sock?secure=True
+func GetRemoteConnection(uri string, identity string) (context.Context, error) {
```

Comment:
Can we consider adding an interface for these features? Podman and Docker have a similar API, and I believe we can leverage that.

Reply:
Ok, good idea

Reply:
Hi! I don't think we need additional packages for this implementation. I'd do:

```
cleaner
- cleaner.go
- podman.go
- docker.go
```